### PR TITLE
Improve error message

### PR DESCRIPTION
**Pull request by NickNewlands**
Story: [AB#13792](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13792)
Task: [AB#13813](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13813)

### CODE REVIEW

File: app/components/general/components/bc-error-page/macro.njk [hunk 1]
```diff
@@ -14,5 +14,10 @@
       <h2 class="nhsuk-heading-2 nhsuk-u-margin-top-5" data-test-id="stackTrace-heading">Stack Trace</h2>
       <p data-test-id="error-stackTrace">{{ params.error.stackTrace | safe }}</p>
     {% endif %}
+    {% if params.error.data %}
+      <pre>
+{{ params.error.data }}
+      </pre>
```
Comment (NickNewlands) [inline app/components/general/components/bc-error-page/macro.njk:20]: This line cannot be indented because the indentation will render.

Comment (bjss-chris) [reply app/components/general/components/bc-error-page/macro.njk:20]: Would formatting it as a single line work?

```nunjucks
<pre>{{ params.error.data }}</pre>
```

Comment (NickNewlands) [reply app/components/general/components/bc-error-page/macro.njk:20]: Good point, I will try it.

Comment (NickNewlands) [reply app/components/general/components/bc-error-page/macro.njk]: That does work, sorry for being dumb.

Comment (bjss-chris) [reply app/components/general/components/bc-error-page/macro.njk]: No worries 😃

**Review by bjss-chris — COMMENTED** [review]
Jus thinking it makes sense to merge your library PR and then update the package reference here before merging this PR.

**Comment by bjss-chris** [comment]
> Just thinking it makes sense to merge your library PR and then update the package reference here before merging this PR.

I've approved the Dependabot PRs so you should just need to update your branch with master now.

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**